spec.html

```diff
@@ -47499,11 +47500,12 @@ <h1>Changes to BlockDeclarationInstantiation</h1>
         </emu-alg>
         <p>During BlockDeclarationInstantiation the following steps are performed in place of step <emu-xref href="#step-blockdeclarationinstantiation-initializebinding"></emu-xref>:</p>
```

Comment:
We could probably generate this preceding paragraph for any `emu-alg[replace-step]` nodes to ensure consistency and remove duplication in the source doc.

Reply:
Eh, I don't think it comes up enough to be worth it.